Sources/Benchmark/BenchmarkResult.swift

```diff
@@ -23,3 +23,18 @@ public struct BenchmarkResult {
         self.measurements = measurements
     }
 }
+
+extension BenchmarkResult: Encodable {
+    private enum CodingKeys: String, CodingKey {
+        case name
+        case iterations
+        case realTime = "real_time"
```

Comment:
What's the reason for the underscore?

Reply:
Looks like compatibility with google/benchmark output.

Reply:
@shabalind Correct. If we don't plan to (or can't) provide a real / sys / user time breakdown, we might consider renaming this to just "time".